tests/visualization_tests/test_contour.py

```diff
@@ -69,7 +98,7 @@ def fail_objective(_: Trial) -> float:
     study = create_study()
     study.optimize(fail_objective, n_trials=1, catch=(ValueError,))
     figure = plot_contour(study, params=params)
-    assert len(figure.data) == 0
+    save_static_image(figure)
```

Comment:
I think we can do away with this test because this test is backend-independent, i.e., we can simply test that the result of `_get_contour_info` is the same as an empty study.

Reply:
This case seems to be already caught in `test_get_contour_info_ignored_error`, so I think we can simply remove this test.